src/sentry/south_migrations/0487_backfill_integrationfeatures.py

```diff
@@ -0,0 +1,1423 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        SentryApp = orm['sentry.SentryApp']
+        IntegrationFeature = orm['sentry.IntegrationFeature']
+
+        sentry_apps = SentryApp.objects.filter(date_deleted=None)
+
+        for app in sentry_apps:
+            IntegrationFeature.objects.create(sentry_app=app)
```

Comment:
This also needs a similar `try...with transaction... except IntegrityError` pattern.